packages/consumption/src/modules/common/ValidationResult.ts

```diff
@@ -21,7 +21,13 @@ export abstract class ValidationResult {
 
     public static fromItems(items: ValidationResult[]): ValidationResult {
         return items.some((r) => r.isError())
-            ? ValidationResult.error(new ApplicationError("inheritedFromItem", "Some child items have errors."), items)
+            ? ValidationResult.error(
+                  new ApplicationError(
+                      "error.consumption.validation.inheritedFromItem",
+                      "Some child items have errors. If this error occurred during the creation of a request, call 'canCreate' to get more information."
```

Comment:
I'm fine with the first two points but the last one would be too much because when you call accept or reject and canAccept or canReject won't be successful there will be thrown: "Cannot accept the Request with the given parameters. Call 'canAccept' to get more information." or "Cannot reject the Request with the given parameters. Call 'canReject' to get more information."